source/Halibut.TestUtils.Contracts/IComplexObjectService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+
+namespace Halibut.TestUtils.Contracts
+{
+    public interface IComplexObjectService
+    {
+        ComplexResponse Process(ComplexRequest request);
+    }
+
+    public class ComplexRequest
+    {
+        public string RequestId;
+        public DataStream Payload;
+        public int NumberOfCopies;
+
+        public ComplexObjectChild Child;
+    }
+
+    public class ComplexResponse
+    {
+        public string RequestId;
+        public IList<DataStream> Payloads;
+
+        public ComplexObjectChild Child;
```

Comment:
This could be made more complex with `Map<Guid, Set<ComplexObjectChild>>` or `Map<Guid, Map<Enum, Set<ComplexObjectChild>>>`

Reply:
Oh `sed s/Map/Dict/` :stuck_out_tongue: